Sources/Imaging/Microsoft.Psi.Imaging.Linux/DepthImageCompressor.cs

```diff
@@ -28,6 +28,9 @@ public DepthImageCompressor(DepthCompressionMethod depthCompressionMethod)
                 case DepthCompressionMethod.Png:
                     this.encoder = new DepthImageToPngStreamEncoder();
                     break;
+                case DepthCompressionMethod.Tiff:
+                    // Currently not supported in Linux.
+                    break;
```

Comment:
I would throw a `NotImplementedException` here to make it clear to the user that it isn't implemented yet.

Reply:
Changed to throwing `NotImplementedException`.